yarn.lock

```diff
@@ -0,0 +1,5756 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
They cause no problems with anything else, but having one does imply that someone would keep them up to date by running `yarn install`.  I have seen them committed frequently in contrib projects within our community (and not just those I maintain).   I would suggest that we bring it up in the next architecture meeting to get wider input.

cc: @amb26